### ISSUE TITLE
V4.1.3 always wants to install additional dependencies

### ISSUE THREAD

**Issue by ndberg** [JeffreyWay/laravel-mix#2216]
-   Laravel Mix Version: 4.1.3
-   Node Version (`node -v`): 10.15.3
-   NPM Version (`npm -v`): 6.9.0
-   OS: ubuntu 18.04.2 (laravel/homestead)

### Description:
Since version 4.1.3, when calling npm run dev, laravel mix always wants to install additional dependencies, without telling which ones. It finishes, but it's en endless loop.

output:
```
vagrant@fotogecko:~/code$ npm run dev

> @ dev /home/vagrant/code
> npm run development


> @ development /home/vagrant/code
> cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js

        Additional dependencies must be installed. This will only take a moment.
 
        Running: npm install  --save-dev --production=false
 
npm WARN browser-sync-webpack-plugin@2.0.1 requires a peer of webpack@^1 || ^2 || ^3 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.0.7 (node_modules/sass/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.0.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

        Okay, done. The following packages have been installed and saved to your package.json dependencies list:
 
        Additional dependencies must be installed. This will only take a moment.
 
        Running: npm install  --save-dev --production=false
 
npm WARN browser-sync-webpack-plugin@2.0.1 requires a peer of webpack@^1 || ^2 || ^3 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.0.7 (node_modules/sass/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.0.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

        Okay, done. The following packages have been installed and saved to your package.json dependencies list:
 
        Finished. Please run Mix again.

```

### Steps To Reproduce:
npm run dev

**Comment by arissberg** [comment]
Same problem

**Comment by mdrebing** [comment]
Same issue here. Also with run dev and watch. Probably caused by [this commit](https://github.com/JeffreyWay/laravel-mix/commit/e23b05a1650dfb44b7552b882fe1931fd64dda1c#diff-018ca3604b99544862ec091b5a005583) from July 16th. Using 4.1.2 for now.

**Comment by wassim** [comment]
Same

**Comment by astefas** [comment]
here as well

**Comment by slidenerd** [comment]
Quadruple this

**Comment by bmadigan** [comment]
It started this morning for me as well but I'm **_not_** on the latest version. Here's my versions:

`node - 10.15.3`
`npm - 6.4.1`
`laravel-mix - 4.0.7`
`osx - 10.14.3 `

I even tried making my `webpack.mix.js` file as minimal as possible but still get the same errors.

**Comment by JeffreyWay** [comment]
This must have been related to the collection rewrite. Patched just now in 4.1.4. Apologies!

**Comment by ndberg** [comment]
thanks
